src/QsCompiler/DllPackaging/Program.cs

```diff
@@ -0,0 +1,203 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.IO;
+using System.Linq;
+using McMaster.Extensions.CommandLineUtils;
```

Comment:
We should stick to using one tool for command line parsing. But I suspect that will change anyway, so I suggest to leave it for now, and then I'll refactor this.